packages/@icon-magic/build/test/fixtures/nav-icons/home/filled.svg

```diff
@@ -0,0 +1,11 @@
+<svg
+  id="Layer_1"
+  data-name="Layer 1"
+  xmlns="http://www.w3.org/2000/svg"
+  width="32"
+  height="32"
+  viewBox="0 0 32 32"
+>
+  <title>32dp</title>
+  <path d="M28,13.36L16.64,6.19a1.2,1.2,0,0,0-1.28,0L4,13.34l1,1.59,2-1.25V25a1,1,0,0,0,1,1h6V21h4v5h6a1,1,0,0,0,1-1V13.67L27,15Z" />
+</svg>
```

Comment:
Where did these svg fixtures come from? Did we create them?

Reply:
This is one of the input icons for the current icon-magic. The designers export these assets from illustrator.